# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: accessible/mac/mozTextAccessible.mm
# Commit: 9bc12053de5e
# Full Hash: 9bc12053de5e6b071d41c81d82859d1567c80c42
# Author: Eitan Isaacson <eitan@monotonous.org>
# Date: 2020-07-03 03:56:55
# Regressor Bug: 1649575
# File Overlap Count: 2
# Description:
#   Bug 1649575 - Part 4: Introduce GeckoTextMarker and GeckoTextMarkerRange. r=morgan
#   
#   These classes do a few things:
#   1. They implement and abstract common text operations, for example:
#    (a) comparing two markers to know which one precedes the other.
# ==============================================================================

diff -r 0441d1553b6b -r 9bc12053de5e accessible/mac/mozTextAccessible.mm
--- a/accessible/mac/mozTextAccessible.mm	Thu Jul 02 21:33:07 2020 +0000
+++ b/accessible/mac/mozTextAccessible.mm	Thu Jul 02 21:35:05 2020 +0000
@@ -5,13 +5,15 @@
 
 #include "Accessible-inl.h"
 #include "HyperTextAccessible-inl.h"
+#include "mozilla/a11y/PDocAccessible.h"
+#include "nsCocoaUtils.h"
+#include "nsIPersistentProperties2.h"
+#include "nsObjCExceptions.h"
 #include "TextLeafAccessible.h"
 
-#include "nsCocoaUtils.h"
-#include "nsObjCExceptions.h"
-
 #import "mozTextAccessible.h"
 
+using namespace mozilla;
 using namespace mozilla::a11y;
 
 inline bool ToNSRange(id aValue, NSRange* aRange) {
